src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -33,7 +33,6 @@
 /** Handles fetching and saving {@link Message} instances. */
 @WebServlet("/messages")
 public class MessageServlet extends HttpServlet {
-
```

Comment:
Did you intend to remove this blank line?

Reply:
Thats typically how I do it, but if I should keep it then I can change it!